src/app/pages/ApproveApp/components/approve-app/approve-app.component.ts

```diff
@@ -29,6 +32,8 @@ export class ApproveAppComponent implements OnInit {
     updateApp(app) {
         this.appService.approveApp(app.appdetail.id)
             .then(success => console.log('success?', success))
+
+        this.addAppComponentRef.nativeElement.scrollIntoView({behavior: "smooth", block:"start"});
```

Comment:
Just noticed that the bug says you should scroll when you select an app, not when you approve an app. When you select an app it does nothing, because updateApp is not called.